docs/orchestration/docker-compose/README.md

```diff
@@ -43,9 +43,6 @@ Each instance is now accessible on the host at ports 9001 through 9004, proceed
 * Minio services in the Docker compose file expose ports 9001 to 9004. This allows multiple services to run on a host.
 
 ### Explore Further
+- [Overview of Docker Compose](https://docs.docker.com/compose/overview/)
+- [Minio Docker Quickstart Guide](https://docs.minio.io/docs/minio-docker-quickstart-guide)
```

Comment:
Minio Docker Swarm doc link can be added here. https://docs.minio.io/docs/deploy-minio-on-docker-swarm